extension/chrome/settings/modules/my_key.ts

```diff
@@ -50,27 +51,27 @@ Catch.try(async () => {
   $('.show_when_showing_public').css('display', '');
   $('.show_when_showing_private').css('display', 'none');
 
-  $('.action_download_pubkey').click(Ui.event.prevent('double', () => {
-    Browser.saveToDownloads(Att.keyinfoAsPubkeyAtt(primaryKi), Catch.browser().name === 'firefox' ? $('body') : undefined);
+  $('.action_download_pubkey, .action_download_prv').click(Ui.event.prevent('double', (e) => {
+    const keyStr = e.className.includes('pubkey') ? primaryKi.public : primaryKi.private;
+    const keyAtt = new Att({ data: Buf.fromUtfStr(keyStr), type: 'application/pgp-keys', name: `0x${primaryKi.longid}.private.asc` });
```

Comment:
And this is actually an error, it would always say `.private.asc` at the end, even for public key, which may lead to confusion.